new_tx.go

```diff
@@ -63,7 +66,12 @@ func (c *Client) OnNewTx(ctx context.Context, params *NewTxParams, callbackFunc
 		callbackFunc(ctx, err, result.(*NewTxNotification))
 	}
 
-	return c.handler.Subscribe(ctx, types.NewTxsFeed, params, wrap)
+	feedType := types.NewTxsFeed
```

Comment:
I recommend creating a new `pending_tx.go` file with all the related functionality. I understand that code is very similar in types.NewTxsFeed and types.PendingTxsFeed, but the params might change in the future, and that would cause additional changes.

Reply:
Yes, I updated this PR. Please take a look